cmd/headscale/cli/routes.go

```diff
@@ -64,28 +59,39 @@ var listRoutesCmd = &cobra.Command{
 		defer cancel()
 		defer conn.Close()
 
-		request := &v1.GetMachineRouteRequest{
-			MachineId: machineID,
-		}
+		var routes []*v1.Route
 
-		response, err := client.GetMachineRoute(ctx, request)
-		if err != nil {
-			ErrorOutput(
-				err,
-				fmt.Sprintf("Cannot get nodes: %s", status.Convert(err).Message()),
-				output,
-			)
+		if machineID == 0 {
```

Comment:
Why is `0` special?

Reply:
oh is zero when you didnt pass the flag?

Reply:
yes!